KestrelMock/Settings/HttpMockSetting.cs

```diff
@@ -1,8 +1,10 @@
-﻿namespace KestrelMockServer.Settings
+﻿using System;
+
+namespace KestrelMockServer.Settings
 {
     public class HttpMockSetting
     {
-        public string Id { get; set; }
+        public string Id { get; set; } = Guid.NewGuid().ToString();
```

Comment:
This will initialize the Id as an actual value, but allows overriding it. It allows us to safely use a ConcurrencyDictionary as the list of items and use the Id as the Key.